src/quo2/components/profile/select_profile/style.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.profile.select-profile.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [customization-color selected?]
+  {:padding          12
+   :background-color (colors/custom-color customization-color 50 40)
+   :border-width     1
+   :border-radius    16
+   :flex             1
+   :border-color     (if selected? colors/white-opa-40 :transparent)
+  })
```

Comment:
We usually have closing brackets in the same line.
@yqrashawn should this break lint?

Reply:
Won't break lint currently. 
zprint don't support this yet. 
But we do want no trailing parens

Reply:
I have updated the formatting manually.